modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/request.json

```diff
@@ -3,1080 +3,1093 @@
+                "example": "12345"
+            },
+            "zipLastFour": {
+                "description": "New zipcode (Last 4 digits) Veteran resides in.",
+                "type": "string",
+                "pattern": "^\\d{4}?$",
+                "example": "6789"
+            },
+            "state": {
+                "description": "New state Veteran resides in.",
+                "type": "string",
+                "pattern": "^[a-z,A-Z]{2}$",
+                "example": "OR"
+            }
+        },
+        "allOf": [
+            {
+            "if": {
+                "properties": {
+                "type": {
+                    "const": "DOMESTIC"
+                }
+                }
+            },
+            "then": {
+                "required": [
+                "city",
+                "zipFirstFive",
+                "state"
+                ]
+            }
+            },
+            {
+            "if": {
+                "properties": {
+                "type": {
+                    "const": "MILITARY"
+                }
+                }
+            },
+            "then": {
+                "required": [
+                "zipFirstFive"
+                ]
+            }
+            }
+        ]
+        },
+        "homeless": {
+            "description": "Object describing Veteran Homelessness if applicable.",
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "pointOfContact": {
+                    "description": "Point of contact in direct contact with Veteran.",
+                    "type": "string",
+                    "minLength": 1,
+                    "maxLength": 100,
+                    "pattern": "^([-a-zA-Z0-9/']+( ?))*$",
+                    "example": "Jane Doe"
+                    },
+                "pointOfContactNumber": {
+                    "telephone": {
+                        "description": "Primary phone of Point of Contact.",
+                        "type": "string",
+                        "example": "1231234",
+                        "maxLength": 10
+                        },
+                    "internationalTelephone": {
+                        "description": "international phone of Point of Contact.",
+                        "type": "string",
+                        "example": "1231234",
+                        "maxLength": 10
+                    }
                 },
-                "city": {
-                  "description": "City Veteran resides in.",
-                  "type": "string",
-                  "pattern": "^([-a-zA-Z0-9'.#]([-a-zA-Z0-9'.# ])?)+$",
-                  "example": "Portland"
+                "currentlyHomeless": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "required": ["homelessSituationOptions", "otherDescription"],
+                    "properties": {
```

Comment:
From the ticket: "Update 'homelessSituationOptions' and 'otherDescription' to be not required."
So `otherDescription` needs to be removed from the required array.